integration_tests/src/testcases/locator.test.ts

```diff
@@ -0,0 +1,403 @@
+import { describe, expect, it, beforeAll, afterAll } from 'vitest';
+import { DropLocator, DropFactory } from '../generated/contractLib';
+import {
+  QueryClient,
+  StakingExtension,
+  BankExtension,
+  setupStakingExtension,
+  setupBankExtension,
+  SigningStargateClient,
+} from '@cosmjs/stargate';
+import { join } from 'path';
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
+import { SigningCosmWasmClient } from '@cosmjs/cosmwasm-stargate';
+import { Client as NeutronClient } from '@neutron-org/client-ts';
+import { AccountData, DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
+import { GasPrice } from '@cosmjs/stargate';
+import { setupPark } from '../testSuite';
+import fs from 'fs';
+import Cosmopark from '@neutron-org/cosmopark';
+
+const DropFactoryClass = DropFactory.Client;
+const DropLocatorClass = DropLocator.Client;
+
+const UNBONDING_TIME = 360;
+
+describe('Locator', () => {
+  const context: {
+    park?: Cosmopark;
+    neutronWallet?: DirectSecp256k1HdWallet;
+    gaiaWallet?: DirectSecp256k1HdWallet;
+    account?: AccountData;
+    neutronClient?: InstanceType<typeof NeutronClient>;
+    neutronUserAddress?: string;
+    client?: SigningCosmWasmClient;
+    gaiaClient?: SigningStargateClient;
+    gaiaQueryClient?: QueryClient & StakingExtension & BankExtension;
+    contracts: {
+      locator?: InstanceType<typeof DropLocatorClass>;
+      factories?: InstanceType<typeof DropFactoryClass>[];
+    };
+    codeIds: {
+      core?: number;
+      token?: number;
+      locator?: number;
+      withdrawalVoucher?: number;
+      withdrawalManager?: number;
+      strategy?: number;
+      staker?: number;
+      puppeteer?: number;
+      validatorsSet?: number;
+      distribution?: number;
+      rewardsManager?: number;
+      factory?: number;
+    };
+  } = { contracts: {}, codeIds: {} };
+
+  beforeAll(async (t) => {
+    context.park = await setupPark(
+      t,
+      ['neutron', 'gaia'],
+      {
+        gaia: {
+          genesis_opts: {
+            'app_state.staking.params.unbonding_time': `${UNBONDING_TIME}s`,
```

Comment:
no need to use this short unbonding time period in this test. Feel free to remove it.